sale_coupon_auto_refresh/models/sale_order.py

```diff
@@ -15,21 +16,31 @@ class SaleOrder(models.Model):
 
     @api.model_create_multi
     def create(self, vals_list):
-        orders = super().create(vals_list)
+        if self.env.context.get("skip_auto_refresh_coupons"):
```

Comment:
One thing that we can check here as well is the context for `website_id` (and there's no need to declare it as a dependency), as `website_sale_coupon` already autorefresh coupons on every `cart_update` and every checkout controller reload, so no need to add an extra load.